src/collections/_documentation/workflow/releases/release-automation/circleci/index.md

```diff
@@ -0,0 +1,74 @@
+
+For more details about scopes and API endpoints, see the full documentation on [API Permissions]({%- link _documentation/api/permissions.md -%}).
+
+[{% asset releases/circleci/internal-integration-permissions.png alt="View of internal integration permissions." %}]({% asset releases/jenkins/internal-integration-permissions.png @path %})
+
+Click “Save” at the bottom of the page and grab your token, which you'll need this in the next step.
+
+## Setting Environment Variables in CircleCI
+
+Next, you'll need a few environment variables to configure the Sentry CLI:
+
+- `SENTRY_AUTH_TOKEN` - Your internal integration token.
+- `SENTRY_ORG` - Your Sentry organization slug.
+- `SENTRY_PROJECT` - Your Sentry project slug.
+
+To access you internal integration token securely, store it as an [environment variable on your project](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project):
+
+1. In the CircleCI application, go to your project’s settings by clicking the gear icon on the Pipelines page, or the three dots on other pages in the application.
+2. Click on Environment Variables.
+3. Add a new variable by clicking the Add Variable button and enter `SENTRY_AUTH_TOKEN` as the name and your internal integration token as the value.
+
+[{% asset releases/circleci/project-settings-env-var.png alt="View of CircleCI environment variables." %}]({% asset releases/circleci/project-settings-env-var.png @path %})
+
+The other variables do not contain sensitive information, so you'll use the environment key to define them as environment variables in your config.yml file.
+
+## Create Release and Notify Sentry of Deployment
+
+In order to automate your Sentry release management process you'll need to add the `Create release and notify Sentry of deploy` step below after your deploy step, either in a new job or at the end of your existing deploy job:
+
+```yaml
+notify-sentry-deploy:
+    executor:
+      # Specify executor for running deploy job
+    environment:
+      SENTRY_ORG: sample-organization-slug
+      SENTRY_PROJECT: sample-project-slug
+      SENTRY_ENVIRONMENT: production
+    steps:
+      - checkout
+# ...
+      - run:
+          name: Create release and notify Sentry of deploy
+          command: | 
+            curl -sL https://sentry.io/get-cli/ | bash
+            export SENTRY_RELEASE=$(sentry-cli releases propose-version)
+            sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
+            sentry-cli releases set-commits $SENTRY_RELEASE --auto
+            sentry-cli releases files $SENTRY_RELEASE upload-sourcemaps build/static/js
+            sentry-cli releases finalize $SENTRY_RELEASE
+            sentry-cli releases deploys $SENTRY_RELEASE new -e $SENTRY_ENVIRONMENT
+```
+
+For more details about the release management concepts in the snippet above, see the full documenation on [release management]({%- link _documentation/cli/releases.md -%}).
+
+**Notes**:
```

Comment:
Worth mentioning that if you're using the webpack plugin for JS, you only need to do the deploy step?